src/resources/views/components/modals/follower.blade.php

```diff
@@ -4,33 +4,69 @@ class="fixed left-0 top-0 bg-mydark bg-opacity-50 w-full h-full justify-center i
         class="bg-mycream rounded-lg shadow-md p-10 flex justify-center items-center">
         <div id="followerModal" class="bg-mycream mx-auto overflow-hidden transition-opacity duration-500">
             <div class="text-mydark font-bold text-center w-full mb-2">
-                <h1>People that followed you</h1>
+                <h1>Followers</h1>
             </div>
+
+            @php
+                $followerIds = $user->followers()->where('status', 1)->pluck('follower_id')->toArray();
+                $followers = App\Models\User::whereIn('id', array_diff($followerIds, [$user->id]))->get();
+            @endphp
+
```

Comment:
I noticed that you always use explicit variables like this. Much better to use mutations for this and then $append = [] so that if you ever need to call that user(model), you can use the appended methods that return some data based on your desired results.

Reply:
I will use $appends, sir